classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt

```diff
@@ -0,0 +1,64 @@
+##### AWS LAB EXECUTION #####
```

Comment:
file is in the wrong folder. it should be your github username.

Reply:
1) how did you connected to the first public instance; This instance I just select public subnet to be available on public IP by auto. 
2) private IP of the instance on private subnet; I select private network with no public IP after install. After that I add a new interface on my public VM with the same subnet on my private subnet to create a connection between this two VMs. 
3)how did you connected from public to private: Add new interface on public machine to talk at same subnet with my private machine is connected.

Reply:
I need you to add the information to the file and update it

Reply:
Cool I´m just check if I´m on good way hehehe. If it is correct now I will do this right now!!!